src/desktop/apps/collect/client.js

```diff
@@ -52,6 +52,8 @@ export const onFilterChange = filters => {
     if (params.medium) {
       route = "/collect" + (params.medium !== "*" ? "/" + params.medium : "")
       delete params.medium
+    } else {
```

Comment:
medium is missing from `params` entirely (as opposed to being set to `*`), so I think that's what the issue was. For purposes of filtering `*` and `null` are going to be treated exactly the same in MP (ie- it will leave it out): https://github.com/artsy/metaphysics/blob/68066aa448dea9d7044500cd26cd98f201a7e68b/src/schema/filter_artworks.ts#L360-L364